.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Jasmine-Node Debugging",
+            "cwd": "${workspaceRoot}",
+            "program": "${workspaceRoot}/node_modules/jasmine/bin/jasmine.js",
+            "request": "launch",
+            "type": "node",
+            "args": [
+                "--config=./spec/support/jasmine.json",
+                "./spec/index.spec.mjs"
+            ]
+        }
+    ]
+}
```

Comment:
missing blank line

Reply:
I've made my contribution, you can update it yourself for formatting